pkg/api/schemas/chat.go

```diff
@@ -17,7 +17,7 @@ func NewChatFromStr(message string) *ChatRequest {
 		Message: ChatMessage{
 			"human",
```

Comment:
I think this needs to be "user" instead of "human"

Reply:
Since all providers take "user" as input in request schema

Reply:
I see yeah it makes sense to change that. However, it makes almost no effect as it's used in tests only to mock request schema. I will take care of this in a followup PR 👀